st2client/st2client/commands/keyvalue.py

```diff
@@ -70,13 +70,17 @@ def __init__(self, *args, **kwargs):
                                                    ' provided prefix.'))
         self.parser.add_argument('--decrypt', action='store_true',
                                  help='Decrypt secrets and display plain text.')
+        self.parser.add_argument('-s', '--scope', default='system', dest='scope',
+                                 help='Scope variable is under. Example: "user".')
 
     def run_and_print(self, args, **kwargs):
         if args.prefix:
             kwargs['prefix'] = args.prefix
 
         decrypt = getattr(args, 'decrypt', False)
         kwargs['params'] = {'decrypt': str(decrypt).lower()}
+        scope = getattr(args, 'scope', 'system')
```

Comment:
Maybe want to use constant here?

(I know we can't re-use st2common stuff here...)